include/CppUTest/CppUTestConfig.h

```diff
@@ -176,6 +176,24 @@
 #endif
 #endif
 
+/* Support for "long long" type; can be enabled or disabled externally or by compiler/platform detects. */
+#if !defined(CPPUTEST_USE_LONG_LONG) && !defined(CPPUTEST_NO_LONG_LONG)
+#if defined(_MSC_VER)
+/* "long long" is always supported without warnings on MSVC */
+#define CPPUTEST_USE_LONG_LONG
+#elif defined(HAVE_LONG_LONG_INT)
+/* config.h says the compiler supports it */
```

Comment:
`config.h` does get created if you run `configure`.  For other build environments it will skip that part and go to the C++11 check below, since any C++11 compiler is supposed to include it.

As currently implemented, the `HAVE_LONG_LONG_INT` check is only really "safe" via configure, as it also adds `-Wno-long-long` to avoid GCC warning spam if `long long` is supported by the compiler but you're compiling pre-C++11, when it was made official.  I don't know if CMake has an equivalent.